Timer

Danwhy opened the pull request:
Adds basic timer functionality:
* Counts down from 30 minutes
* Able to pause

ref #7

Danwhy committed:
adds timer to details page, #7

Danwhy committed:
Merge branch 'fs' into timer

sohilpandya commented:
Hey @Danwhy, could you please invite me to the doc that contains the env file. I'm unable to run the server, which leads to errors in UI. :(

Danwhy committed:
adds instructions for running dyanamodb locally

Danwhy commented:
Sorry @sohilpandya  I forgot to include instructions to make the table locally. I've updated the readme with instructions and the env variables you should need. It might also be easier to run the app from the `heroku` branch (#21), where I've added the `start:dev` script

sohilpandya commented:
Great! I'll merge this one in and leave some comments in the `heroku` branch! :)
